### ISSUE TITLE
[BUG] Disable shadows on hairs? 

### ISSUE THREAD

**Issue by lab-404-development** [IamPerek/prk-fivem-appearance#1]
you fix this bug?
![image](https://user-images.githubusercontent.com/82737979/156921005-bd9a12eb-c113-4d3a-addb-1f4023e83de5.png)

**Comment by IamPerek** [comment]
> you fix this bug? ![image](https://user-images.githubusercontent.com/82737979/156921005-bd9a12eb-c113-4d3a-addb-1f4023e83de5.png)

Did you do everything that is written in README.md?

**Comment by IamPerek** [comment]
> you fix this bug? ![image](https://user-images.githubusercontent.com/82737979/156921005-bd9a12eb-c113-4d3a-addb-1f4023e83de5.png)

the function to disable shadows is not there yet I will try to add it

**Comment by lab-404-development** [comment]
> > you fix this bug? ![image](https://user-images.githubusercontent.com/82737979/156921005-bd9a12eb-c113-4d3a-addb-1f4023e83de5.png)
> 
> Did you do everything that is written in README.md?

yes i did but i have that problem. shadows for customs hair only.

**Comment by lab-404-development** [comment]
you found solution for that?

**Comment by TweakyGaming** [comment]
That's only when you create. As soon as you save, the textures disappear again. I would also prefer that it is saved it looks so much better. But it may well be that there are problems with custom clothing.

**Comment by IamPerek** [comment]
> That's only when you create. As soon as you save, the textures disappear again. I would also prefer that it is saved it looks so much better. But it may well be that there are problems with custom clothing.

yea but i think that all fivem-appearance for QBCore have this bug so its not only mine :)
